### PR TITLE
Footer Data Formatting

### PR DESCRIPTION
**Pull request by sumitd94**
This is fix for PR #16

### CODE REVIEW

File: components/footer/footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.infoRepo {
+  font-weight: 100;
+  margin: 0 auto;
+  text-align: center;
+}
```
Comment (invalidtoken) [inline components/footer/footer.module.scss:5]: Styles Looks Good.

File: components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <p className={classNames.infoRepo}>
+        The contents of this website are deployed from this{' '}
+        <a
+          href="https://github.com/Real-Dev-Squad/website-members"
+          target="_blank"
+          rel="noopener noreferrer">
+          open sourced repo
+        </a>
+      </p>
+    </footer>
```
Comment (invalidtoken) [inline components/footer/index.js:16]: Good, the code is formatted properly.

File: components/footer/footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.infoRepo {
+  font-weight: 100;
+  margin: 0 auto;
+  text-align: center;
```
Comment (invalidtoken) [inline components/footer/footer.module.scss:4]: Just add top and bottom padding of 15px.

Comment (sumitd94) [reply components/footer/footer.module.scss:4]: Done, Could you please check

**Pull request merged**